### ISSUE TITLE
Fix tag colors and style

### ISSUE THREAD

**Issue by whoabuddy** [AbsorbingChaos/unmuteable-com#60]
Current tags fail the WCAG accessibility standard for legible text. The mint color with white text doesn't have a high enough contrast.

Possible solution: change tags to black and add mint border?
